docs/reference.md

```diff
@@ -64,6 +64,65 @@ Ready message should be send by a plugins as soon as it renders their content an
 
 If ready message will not be sent within ~10 seconds, Ellipsis will render a timeout error and plugin content will be discarded.
 
+Request additional Story ANS
+
+Customers can obtain addition ANS data from Ellipsis via `postMessage`. Ellipsis/Composer will send back the data with the additional fields of:
+
+1. Headlines
+2. Subheadlines
+3. Taxonomy
+
+To acheive this:
+- Add an additional flag `isAnsRequired` set to `true` in your API where `postMessage()` is happening.
+- `postMessage()` with Ready message `action = 'ready'`
+- E.g.
+```javascript
+  window.parent.postMessage(
+    JSON.stringify({
+      source: 'custom_embed',
+      action: 'ready',
+      data: { height: 900 },
+      key: parseQueryString()['k'],
+      isAnsRequired: false    // Change this to true
+    }),
+    targetOrigin
+  )
+```
+- Which will send out a message to Ellipsis/Composer that looks like:
+```javascript
+{source: "custom_embed", action: "ready", data: {…}, key: "1234567890", isAnsRequired: true}
+```
+- Ellipsis/Composer will postMessage back with additional Story ANS data:
+```javascript
+MessageEvent 
+{
+   "message":"ans_data",
+   "data":{
+      "headlines":{
+         "basic":"Test Headline",
+         "mobile":"",
+         "native":"",
+         ...
+      },
+      "subheadlines":{  "basic":"Test Sub Headline"},
+      "taxonomy":{
+         "sites":[{"_id":"/about-us", "type":"site", ...}],
+         "tags":[],
+         "sections":[],
+         ...
+      }
+   }
+}
+```
+#### Note
+- Ellipsis/Composer can send ANS back to Search, View, and Edit API
```

Comment:
could u change the `API` to integration to match the phase used at other places?